### DISCUSSION TITLE
بلاک بودن سایت و اپ بانک

### DISCUSSION THREAD

**Discussion by ne7worker** [hiddify/Hiddify-Manager#312] [Bug باگ: please use issue]
سلام، وقت بخیر. روی سرویس من حتی وقتی سایتهای ایرانی رو آنبلاک میکنم سایت بانک ها و اپ بانکها باز نمیشن

**Comment by hiddify-com** [comment]
خوب اینکه طبیعیه سایتهای بانک و اینا از آی پی خارج نمیتونن وصل بشن

**Reply by ne7worker** [reply]
.ممنونم که جواب دادید. بله در حالت عادی درسته وقتی هیچکدوم باز نشه، ولی مثلا شاپرک برای پرداخت باز میشه، یا آپ. فکر میکردم جایی بشه استثنا تعریف کرد برای عبور

**Comment by hiddify-com** [comment]
میتونی لیست سایت ها و دامنه های بانک ها را بهم بدی دستی ست کنیم؟

**Reply by ne7worker** [reply]
ببین مثلا parsian-bank.ir
https://ipb.parsian-bank.ir/login.html
bmi.ir
https://www.rb24.ir/login.html

اینم بگم که کار شما خیلی عالیه و دمتون گرم، یعنی این دراصل یه آپشنه که میتونستین نذارین، این بیشتر suggestion حساب میشه

**Comment by old-joker** [comment]
i think we can add this 
https://github.com/v2fly/domain-list-community
https://github.com/Loyalsoldier/v2ray-rules-dat/

GEOIP.dat file 
and a regex for .ir site 

`"routing": {
  "rules": [
    {
      "type": "field",
      "outboundTag": "Direct",
      "ip": [
        "geoip:ir",
        "geoip:private"
      ]
    },
       {
      "type": "field",
      "outboundTag": "Direct",
      "domain": [
        "geosite:ir",
        "geosite:private"
      ]
    },
  ]
}`
